drivers/usb/dwc3/core.c

```diff
@@ -566,6 +566,11 @@ static int dwc3_phy_setup(struct dwc3 *dwc)
 		ret = dwc3_ulpi_init(dwc);
 		if (ret)
 			return ret;
+		if(dwc->dr_mode == USB_DR_MODE_HOST)
+		{
+                        reg |= DWC3_GUSB2PHYCFG_ULPI_EXT_VBUS_DRV;
```

Comment:
I didn't compile against the stock ULPI driver.
 
# CONFIG_USB_ULPI_BUS is not set

Are you setting this in .config?